### DISCUSSION TITLE
[FR] Relational field setting for showing entries from sites user does not have permission to edit

### DISCUSSION THREAD

**Discussion by rungta** [craftcms/cms#8815] [Ideas]
### Description

Now that relation fields (that aren’t locked to a single site) allow authors to select entries from different sites (thanks to #3584), it is possible to have cross-site relations (for non-propagating sections). However, an author currently needs Edit permissions for a site for its entries to show up in the element selector modal. It would be great if there was an option to also expose entries from sites the author did not have Edit permissions for, à la “Show unpermitted volumes” and “Show unpermitted files” checkboxes in the Asset fields.

### Steps to reproduce

1. Create a multi-site setup with two sites (_Site A_, _Site B_).
2. Create a section (_News_) that does not propagate entries to other sites.
3. Add some _News_ entries in both _Site A_ and _Site B_.
4. Create an entries field for _News_ and add it to a section, say _Pages_.
5. Create a user with edit permissions for _Site B_ only (and all sections).
6. Login as this user and create a new _Pages_ entry inside _Site B_, try to add some _News_ entries from _Site A_. 

### Additional info

- Craft version: 3.4.14

**Comment by rungta** [comment]
Hey @brandonkelly, any insight into the dev timeline (3.4.x / 3.5 / later) for this would be very helpful for a project we’re currently working on.

**Comment by brandonkelly** [comment]
@rungta Realistically this is more of a 4.0 thing that we can look at when moving permissions over to RBAC (#1646). In the meantime I think you will just need to give users permission to access other sites.

**Comment by rungta** [comment]
Ah okay. Thanks for the heads-up. I was hoping 🤞 the logic/process for showing unpermitted files in Asset selector modals could be availed here as well without introducing any new permission roles.

![image](https://user-images.githubusercontent.com/663677/79151932-f8e71580-7de8-11ea-86cd-0db6d9a6928c.png)

**Comment by brandonkelly** [comment]
Yeah in that case it’s still based on permissions though. So that needs to be the starting point.
